CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -2655,6 +2618,55 @@ public function processConfirm(
     throw new CRM_Core_Exception('code is unreachable, exception is for clarity for refactoring');
   }
 
+  /**
+   * This was extracted from processConfirm() as part of a refactoring process.
+   * This should probably throw the exception instead of catching it and putting it in an array
+   * @internal
+   *
+   * @param \CRM_Contribute_DAO_Contribution $contribution
+   * @param int $contactID
+   *
+   * @return array
+   * @throws \CRM_Core_Exception
+   */
+  private function processConfirmPayment(\CRM_Contribute_DAO_Contribution $contribution, int $contactID): array {
+    $form = $this;
```

Comment:
Should do a find/replace in this function but want to do a straight extraction first.